doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -92,17 +92,25 @@ Bluetooth Mesh
 Matter
 ------
 
+* From |NCS| 2.6.0 version the Arm PSA Crypto API is used as cryptography for devices that use Matter over Thread.
+  You can still build all examples with deprecated mbedTLS support by setting the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA` Kconfig option to ``n``, but you must build Thread library from sources.
+  To use :ref:`inheritance of Thread certification <ug_matter_device_certification_reqs_dependent>` from Nordic Semiconductor you MUST use the PSA Crypto API backend.
 * Updated the page about :ref:`ug_matter_device_low_power_configuration` with the information about Intermittently Connected Devices (ICD) configuration.
 
 * Added:
 
   * A Kconfig option for disabling or enabling :ref:`ug_matter_configuring_read_client`.
-  * Support for PSA Crypto API for devices that use Matter over Thread.
-    It is enabled by default and can be disabled by setting the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA` Kconfig option to ``n``.
   * Migration of the Device Attestation Certificate (DAC) private key from the factory data set to the PSA ITS secure storage.
 
     The DAC private key can be removed from the factory data set after the migration.
     You can enable this experimental functionality by setting the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA_MIGRATE_DAC_PRIV_KEY` Kconfig option to ``y``.
+  * Migration of the Node Operational Key Pair (NOK) from the mbedTLS-related settings storage to the PSA ITS secure storage.
+    All existing NOKs (for all existing Matter fabrics) will be migrated to the PSA ITS secure storage at the boot of the device.
```

Comment:
```suggestion
    All existing NOKs for all Matter fabrics will be migrated to the PSA ITS secure storage at boot.

```
Since only a device boots, it's not necessary to say it's the boot of the device.